gusto/timestepping/semi_implicit_quasi_newton.py

```diff
@@ -850,15 +842,19 @@ def __init__(self, equation, solver_prognostics, implicit_terms,
                 passed in. Defaults to False.
             reference_dependent: this indicates that the solver Jacobian should
                 be rebuilt if the reference profiles have been updated.
+            appctx: appctx for the  underlying :class:`LinearVariationalSolver`.
```

Comment:
more doc changes required above to remove obsolete arguments and put docs in same order as args